packages/storybook/.storybook/preview.js

```diff
@@ -104,3 +104,6 @@ export default {
 configureActions({
     depth: 1
 });
+
+// Update the GUID on this line to trigger a turbosnap full rebuild: 3e3470f8-2a6c-4474-b409-409c1dfbecea
+// See https://www.chromatic.com/docs/turbosnap/#full-rebuilds
```

Comment:
@jattasNI added this to manually disable turbosnap for a build and looks like it helped trigger full snapshotting:
https://github.com/ni/nimble/actions/runs/9116910090/job/25066573475?pr=2114#step:16:54
![image](https://github.com/ni/nimble/assets/1588923/8c65f73d-60ad-4193-bc59-ab242d3041b4)

So I think this PR is good. I'll hold off documenting, etc while investigating the behavior. I'll create an issue though.

Reply:
Marked automerge for once resolved (don't think we need to wait for all owners)